test/fixtures/mson-enum.json

```diff
@@ -0,0 +1,49 @@
+{
+  "_version": "3.0",
+  "metadata": [],
+  "name": "<API name>",
+  "description": "<API description>\n\n",
+  "element": "category",
+  "resourceGroups": [],
+  "content": [
+    {
+      "element": "category",
+      "content": [
+        {
+          "element": "object",
+          "meta": {
+            "id": "User",
+            "title": "User"
+          },
+          "content": [
+            {
+              "element": "object",
```

Comment:
I made it according to `tag` example in specs.

But You are right, it seem to be one more level there.

@zdne ?
https://github.com/refractproject/refract-spec/blob/master/namespaces/mson-namespace.md#examples